### PR TITLE
added extra caution for SUSE users, updated script

### PR DESCRIPTION
**Pull request by rachejazz**
Signed-off-by: rachejazz <divyadeepti2000@gmail.com>

Tested my script recently on OpenSUSE, looks like python3 needs to be specified since it has python2 as default.
And also, there was no `/bin/python3` :'D on SUSE
Changed and added a workaround too in case of permission error.

### CODE REVIEW

**Comment by rachejazz** [comment]
I'll update the script as well and then you can review this PR.

**Comment by rachejazz** [comment]
image

File: src/config/config.py [hunk 1]
```diff
@@ -39,6 +39,9 @@
 	},
 	'Debian 10': {
 		'Buster': 'Debian_Buster_List.json'
+	},
+	'ClefOS': {
+		'ClefOS Epel 7': 'ClefOS_7_List.json'
```
Comment (pleia2) [inline src/config/config.py:44]: Epel is a different section of the repository :smile: (we may include it later, based on feedback from Neale) according to the script above, you want "base" here.

File: bin/package_build.py [hunk 2]
```diff
@@ -1,4 +1,4 @@
-#!/bin/python
+#!/usr/bin/python3
```
Comment (pleia2) [inline bin/package_build.py:1]: Yeah, I had to make the same change on Debian. If this works better everywhere :+1:

**Review by pleia2 — CHANGES_REQUESTED** [review]
Just one tiny change to the description in the config file.

**Review by pleia2 — APPROVED** [review]
Looks good, but I'll hold off on actual merging until we hear back from Neale :+1:

**Comment by rachejazz** [comment]
Should I proceed for OpenSUSE addition? Sine Nominie response might delay

**Comment by pleia2** [comment]
I'll just merge this. We can adjust later if there are problems, and we'll still wait on the data import.

**Pull request merged**